### ISSUE TITLE
Cannot decrypt repositories with OpenSSL 1.1.0 and later, that were created with earlier versions of OpenSSL

### ISSUE THREAD

**Issue by ircecho** [elasticdog/transcrypt#40]
The reason is that [default digest for the dgst and enc commands from MD5 to sha256](https://www.openssl.org/news/cl110.txt).

When setting the digest to md5, the files can be decrypted properly again.

**Comment by lynchem** [comment]
Could you share your change @ircecho  ? We have the same problem.

**Comment by ircecho** [comment]
You have to change the following lines in the files in the folder ".git/crypt":

```
clean: ENC_PASS=$password openssl enc -$cipher -md MD5 -pass env:ENC_PASS -e -a -S "$salt" -in "$tempfile"
smudge: tee "$tempfile" | ENC_PASS=$password openssl enc -$cipher -md MD5 -pass env:ENC_PASS -d -a 2> /dev/null || cat "$tempfile"
textconv:  ENC_PASS=$password openssl enc -$cipher -md MD5 -pass env:ENC_PASS -d -a -in "$filename" 2> /dev/null || cat "$filename"
```

See the added **-md MD5**.

**Comment by lynchem** [comment]
Thanks for that @ircecho . All working nicely.

**Comment by thorntonrose** [comment]
This fix worked for me on a Debian 8 system. Thanks. When will this fix be released?

**Comment by elasticdog** [comment]
This has been resolved by #41 and I'm working on cutting a new release as we speak.
